React_pakadmin/tar_19/src/SortableTable.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import "../css/main.css";
+
+export default function SortableTable({ data }) {
+  const columnsEnum = { name: 1, country: 2, email: 3 };
+  const [inputData, setInputData] = useState(data.slice(1, data.length));
+  const [columnsClickCounter, setColmnsClickCounter] = useState({
+    name: 0,
+    country: 0,
+    email: 0,
+  });
+
+  function sortInputData(columnName) {
+    setColmnsClickCounter({
+      ...columnsClickCounter,
+      [columnName]: columnsClickCounter[columnName] + 1,
```

Comment:
Why do you keep the state of clicks for each field? You can just set one variable for ascending/descending order.
Why do you need to remember previous sortings?
This overengineering in my opinion and it does not add anything meaningful but make sit harder to read, understand, and modify later.